db/deploy/computed_columns/application_analyst_lead.sql

```diff
@@ -0,0 +1,19 @@
+-- Deploy ccbc:computed_columns/application_analyst_lead to pg
+
+begin;
+
+create or replace function ccbc_public.application_analyst_lead(application ccbc_public.application) returns varchar as
+$$
+select concat_ws(' ', given_name, family_name) AS name from ccbc_public.application_analyst_lead as al
```

Comment:
I was going back and forth on whether I think this should return the whole row, and I think it would be more useful in future cases to return the row if we want to use it for joins later. Up to you since this is essentially designing for more than the ticket needs